src/SFA.DAS.EmployerIncentives.Api/Controllers/CollectionCalendarCommandController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Api.Types;
+using SFA.DAS.EmployerIncentives.Commands.CollectionPeriod;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerIncentives.Api.Controllers
+{
+    [Route("collectionCalendar")]
+    [ApiController]
+    public class CollectionCalendarCommandController : ApiCommandControllerBase
+    {
+        public CollectionCalendarCommandController(ICommandDispatcher commandDispatcher) : base(commandDispatcher)
+        {
+        }
+
+        [HttpPost]
```

Comment:
POST is used to create data.  We are changing an existing record so should be a PATCH or a PUT to the Periods endpoint.  The "IsActive" state should be in the data along with the period number and year since that is what is being changed.  Otherwise we will end up with an RPC style API which creates extra coupling.

Reply:
I disagree about having the IsActive state in the data, that would require the calling Azure function to retrieve all the collection periods in order to set all the other periods to inactive and set the new period to active. Happy to change the Http method though.

Reply:
The caller wouldn't have to retrieve all other collection periods.  Because IsActive is set on this one means the others will be set to false as a result since there can only be one that IsActive.  By putting it in the data means that we can easily meet future requirements without breaking or changing the interface every time.

Reply:
Updated the API to allow for de-activation of collection periods for the kill switch scenario as discussed.